libsolidity/interface/CompilerStack.h

```diff
@@ -104,6 +104,12 @@ class CompilerStack: boost::noncopyable
 		None
 	};
 
+	enum class VersionType {
+		Release,
+		Prerelease,
+		Empty
+	};
```

Comment:
`VersionType` sounds pretty generic to me and without looking at the implementation, I probably would not guess what it changes about the compilation. How about changing it to something more self-explanatory?

```suggestion
	enum class MetadataFormat {
		WithReleaseVersionTag,
		WithPrereleaseVersionTag,
		NoMetadata,
	};
```

And also changing `setVersionType()` and `m_versionType` to match.